oura-library/src/main/kotlin/org/radarbase/oura/request/OuraRequestGenerator.kt

```diff
@@ -112,7 +132,16 @@ constructor(
                 request.user,
                 Instant.ofEpochSecond(offset).plus(Duration.ofMillis(500)),
             )
-            userNextRequest[request.user.versionedId] = Instant.now().plus(SUCCESS_BACK_OFF_TIME)
+            val nextRequestTime = userNextRequest[request.user.versionedId]
+            userNextRequest[request.user.versionedId] =
+                nextRequestTime?.let {
+                    if (nextRequestTime > Instant.now()) {
```

Comment:
should it check with ` Instant.now().plus(SUCCESS_BACK_OFF_TIME)`?

Reply:
Ah yes made the changes.